BaragonService/src/main/java/com/hubspot/baragon/service/config/PurgeCacheConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package com.hubspot.baragon.service.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hubspot.baragon.models.BaragonService;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PurgeCacheConfiguration {
+  @JsonProperty("enabledTemplates")
+  @NotNull
+  private List<String> enabledTemplates;
+
+  @JsonProperty(value = "excludedServiceIds")
+  @NotNull
+  private List<String> excludedServiceIds;
+
+  public List<String> getEnabledTemplates() {
+    if (enabledTemplates == null) {
```

Comment:
you can actually just set the `private List<String> enabledTemplates = new ArrayList();` above and not have to worry about this. When dropwizard/jackson reads the config it will override based on the yaml if needed (since you also provided a setter)

Reply:
Changed this